### PR TITLE
Djl/d3d inpa

### PR DESCRIPTION
**Pull request by djlin1**
Added INPA geometry on d3d and modified d3d_chords to prevent redundant addition of fibers.

### CODE REVIEW

File: src/d3d_chords.pro [hunk 1]
```diff
@@ -53,14 +53,16 @@ FUNCTION oblique_geo_from_patch, patchfile, skip=skip
         f = patchdat.(i).fibers
         for j=0,n_elements(f.fibers)-1 do begin
             fnum = f.fibers[j]
+            name = 'p'+string(ceil(float(fnum)/3),FOR='(I02)')+suffix[fnum mod 3 -1]
+            IF N_ELEMENTS(id) && TOTAL(STRCMP(name,id)) THEN CONTINUE
```
Comment (lstagner) [inline src/d3d_chords.pro:57]: What is the purpose of this change, again?

Comment (djlin1) [reply src/d3d_chords.pro:57]: Each chord contains information for the whole row of fiber. If two different chords reside on the same row, redundant fiber geometry will be appended. This will check if the fiber information has been already added.
